test/forms_test.js

```diff
@@ -563,9 +563,9 @@ describe('Forms', function() {
       before(function(done) {
         const field1 = browser.querySelector('#field-scary');
         const field2 = browser.querySelector('#field-notscary');
-        browser.choose(field1);
-        field2.addEventListener('focus', ()=> done());
         browser.choose(field2);
+        field1.addEventListener('focus', ()=> done());
+        browser.choose(field1);
       });
```

Comment:
I guess Mocha is working better now and finding bugs in the test suite. I think in this case the correct measure is to remove the listener in an `after` block.